### ISSUE TITLE
Amplify-CLI not working (amplify add auth)

### ISSUE THREAD

**Issue by Rayv1** [aws-amplify/amplify-flutter#73]
**Describe the bug**
I installed the latest Version of ampliy-cli for flutter and i could run amplify-init and so on. But when i try to add auth "amplify add auth" and select something, the cli instantly exit. This also happens when i try to add storage and the cli asks then for auth.

**To Reproduce**
On Windows 10, install the Version 4.26.1-flutter-preview.0 and run "amplify add auth"

**Expected behavior**
adding Auth

**Screenshots**
$PS: xxx > amplify add auth
 Do you want to use the default authentication and security configuration? Default configuration
 Warning: you will not be able to edit these selections.
 How do you want users to be able to sign in? (Use arrow keys)
> Username
  Email
  Phone Number
  Email or Phone Number
  I want to learn more.
$PS: xxx >

The Real Problem is, that i just want to know how the Configuration should look like, because i hate using the cli and will use my already created Cognito User Pool.

Best Regards,
Ray

**Comment by haverchuck** [comment]
@Rayv1 We will take a look at this behavior.  In the meantime, [our documentation shows the expected structure of the config](https://docs.amplify.aws/lib/auth/existing-resources/q/platform/flutter).

**Comment by haverchuck** [comment]
@Rayv1 I've confirmed the functionality on a Windows machine; when you ran into this issue, the CLI crashed after the question `How do you want users to be able to sign in? (Use arrow keys)`?

**Comment by Rayv1** [comment]
@haverchuck yes exactly. I started my Command Line from vscode and it exits when i select one of the first three. When i click on "i want to learn more" its not exiting.
